Resources/Prototypes/_Goobstation/Damage/modifier_sets.yml

```
@@ -32,6 +32,15 @@
     Poison: 0.0 # Yea, heroin is heaps good for ya
     Radiation : 0.7 # Engie yowies need some help against anti-rads
 
+- type: damageModifierSet 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)